WebApp/src/actions/timetableActions.js

```diff
@@ -171,14 +171,14 @@ export function exchangeConfirm(blockTo) {
       {
         courseId: bl.id,
         day: bl.day,
-        startHour: bl.startBlock,
+        startHour: bl.startBlock + 6,
```

Comment:
I'm not javascript developer, but generaly I don't like magic numbers like this - may be we should define somewhere some constatns, or better some function to convert the block identifier to block time...